pages/polls/[id].vue

```diff
@@ -1,13 +1,45 @@
 <template>
     <div>
-
-    </div>
+      <PollItem
+          v-if="poll" :id=poll.id :name=poll.name :description=poll.description :end-date=poll.endDate
+          :upvote-count=poll.upvotes :downvote-count=poll.downvotes
+      />
+    </div>  
 </template>
 
 <script setup>
 
 </script>
 
+<script>
+export default {
+  beforeMount() {
+    this.fetchPoll().then((result) => {
+      this.poll = result;
+      console.log(result)
+      document.title = result.value ? result.value.name : "POLLme";
```

Comment:
```suggestion
document.title = 'POLLmelo - ' + (result.value ? result.value.name : "");
```
i added a POLLmelo infront, I think it is better this way